### PR TITLE
Provide test suites with an instance of `Storage`

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
This PR contains a method to get a `Storage` instance for test.

Also, it contains newly generated Gradle wrapper for version 4.0.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
At this moment, we need these changes also in the `gae-java` module.

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#Tue May 30 16:38:29 EEST 2017
+#Thu Jun 22 12:16:36 EEST 2017
```
Comment (armiol) [inline gradle/wrapper/gradle-wrapper.properties]: I guess the changes to this file may be reverted.

File: gradlew [hunk 2]
```diff
@@ -33,11 +33,11 @@ DEFAULT_JVM_OPTS=""
 # Use the maximum available, or set MAX_FD != -1 to use that value.
 MAX_FD="maximum"
 
-warn ( ) {
```
Comment (armiol) [inline gradlew]: Same as above. Just to make the PR cleaner.

**Pull request merged**